client/src/components/Landing.js

```diff
@@ -1,13 +1,31 @@
 import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import './Landing.css';
 
 class Landing extends Component {
     render() {
         return(
             <div>
-                This is the page you see when you are at our landing page.
+                <div id="background">
+                    <img src={require('../assets/images/landing-temp.jpg')} alt="" />  
+                
+                    <div id="now-playing">
+                        <div class="now-playing-text">
+                            <div id="main-text">You're listening to</div>
+                            <img src={require('../assets/images/album-temp.jpg')} alt="" />
+                            <div class="song-info"> 
+                                <div id="song-title">Song</div>
+                                <div id="artist-name">Artist</div>
+                            </div>
```

Comment:
these IDs don't currently reference any css

Reply:
I'm planning on giving these different text sizes and bold width.